File: practica n6/pagina.py
class Pagina:   
    def __init__(self, numero: int, contenido: str):
        self.numero = numero
        self.contenido = contenido


    def mostrar_pagina(self) -> None:
        print(f"Página {self.numero}: {self.contenido}")

    def __str__(self) -> str:
        return f"Página {self.numero}: {self.contenido[:30]}"

File: practicas/bibli/pagina.java
package practicas.bibli;


public class pagina {
    private int numero;
    private String contenido;

    public pagina(int numero, String contenido) {
        this.numero = numero;
        this.contenido = contenido;
    }

    public void mostrarPagina() {
        System.out.println("Pagina " + numero + ": " + contenido);
    }
}